include/backend/drm/drm.h

```diff
@@ -33,6 +33,7 @@ struct wlr_drm_plane {
 	float matrix[16];
 	struct wlr_texture *wlr_tex;
 	struct gbm_bo *cursor_bo;
+	bool cursor_enabled;
```

Comment:
There already is a bool in wlr_output which checks if it's a hardware or software cursor.

Reply:
This one has a different purpose: it checks whether or not the hardware cursor is enabled. A hardware cursor can be disabled even if software cursors are not used (e.g. when playing a fullscreen video, the cursor is hidden).